waku/node/waku_node.nim

```diff
@@ -1082,16 +1083,29 @@ proc isBindIpWithZeroPort(inputMultiAdd: MultiAddress): bool =
 
 proc printNodeNetworkInfo*(node: WakuNode): void =
   let peerInfo = node.switch.peerInfo
+  var announcedStr = ""
   var listenStr = ""
+  var localIp = ""
+
+  try:
+    localIp = $getPrimaryIPAddr()
+  except Exception as e:
+    warn "Could not retrieve localIp", msg=e.msg
```

Comment:
```suggestion
  let res = catch: getPrimaryIPAddr()
  localIp = $res.valueOr:
    warn "Could not retrieve localIp", msg=error.msg
```
I like `catch:` it makes `try except` more concise and less drift. https://github.com/arnetheduck/nim-results/blob/f3c666a272c69d70cb41e7245e7f6844797303ad/results.nim#L723

Reply:
Ooh that looks better, will do it that way! Thank you!